src/select-menu/src/OptionsList.js

```diff
@@ -48,7 +50,8 @@ export default class OptionsList extends PureComponent {
     filterPlaceholder: PropTypes.string,
     filterIcon: PropTypes.string,
     optionsFilter: PropTypes.func,
-    defaultSearchValue: PropTypes.string
+    defaultSearchValue: PropTypes.string,
+    closeOnSelect: PropTypes.bool
```

Comment:
While the name is pretty semantic would you mind adding some jsdoc description above the new prop?

Reply:
no worries, done